pkg/networkserver/grpc_gsns.go

```diff
@@ -923,8 +923,11 @@ func (ns *NetworkServer) newDevAddr(context.Context, *ttnpb.EndDevice) types.Dev
 
 func (ns *NetworkServer) sendJoinRequest(ctx context.Context, ids ttnpb.EndDeviceIdentifiers, req *ttnpb.JoinRequest) (*ttnpb.JoinResponse, error) {
 	logger := log.FromContext(ctx)
-	if js := ns.GetPeer(ctx, ttnpb.ClusterRole_JOIN_SERVER, ids); js != nil {
-		resp, err := ttnpb.NewNsJsClient(js.Conn()).HandleJoin(ctx, req, ns.WithClusterAuth())
+	cc, err := ns.GetPeerConn(ctx, ttnpb.ClusterRole_JOIN_SERVER, ids)
+	if err != nil && !errors.IsNotFound(err) {
```

Comment:
`errPeerEmptyTarget` is not a `NotFound` but is not reason to exit the whole join flow here. That should gracefully go to interop

Reply:
@rvolosatovs please advise here

Reply:
Indeed, just log the error at error level, do not return here.